dist/RadarSDK.framework/Versions/A/Headers/RadarTrackingOptions.h

```diff
@@ -161,23 +161,23 @@ typedef NS_ENUM(NSInteger, RadarTrackingOptionsSync) {
 @property (nonatomic, assign) BOOL beacons;
 
 /**
- A preset that updates about every 30 seconds and syncs all locations to the server. High battery usage. Shows the flashing blue status bar when tracking.
```

Comment:
a bit unsure about why this text changed within the `dist` folder. Wanted to bring this to attention to see which is correct before I reverted to what was in master for this file

Reply:
I'm guessing this was sitting in `master` but hadn't been released yet. These changes are correct, we should keep them in this release.

Reply:
it looks to me like this PR https://github.com/radarlabs/radar-sdk-ios/pull/118 changed the text, but didnt built. so dist is now being updated to reflect that change